zoo-css-challenge/style.css

```diff
@@ -55,6 +63,10 @@ h3 {
   font-size: 150%;
 }
 
+p {
```

Comment:
Hi Abed, good work, although you may want to consider applying line height to the body element so that the line-height property is applied to all text. You can then override it in places where you need to with more specific css selectors.